content/conditions/constipation/main-content.md

```diff
@@ -0,0 +1,161 @@
+
+To make it easier to poo, try resting your feet on a low stool while going to the toilet. If possible, raise your knees above your hips.
+
+## Babies and toddlers: what causes constipation
+
+Constipation in babies and toddlers has many possible causes. Sometimes there is no obvious reason. 
+
+It usually happens when your child:
+
+- first starts taking formula or processed foods as a baby
+- is being potty trained as a toddler
+- has just started school
+
+The most common causes include:
+
+- not eating enough fibre - such as fruit, vegetables and cereals
+- not drinking enough fluids
+- over-feeding - including giving babies too much milk
+- fear or anxiety about using the toilet - at home or at school
+- poor potty training - such as feeling pressured or being regularly interrupted
+
+In much rarer cases, constipation in babies and toddlers may be caused by a medical condition.
+
+
+## Babies and toddlers: treating constipation
+
+Simple changes to your child’s diet and potty training can help treat constipation.
+
+You may notice a difference within a few days. Sometimes it takes a few weeks before their symptoms improve.
+
+### Make changes to your child’s diet
+
+Give your baby extra water between their normal feeds if they haven’t started to eat solid food yet. If you’re using formula milk, don’t add more water to the mixture.
+
+Try gently moving your baby's legs in a bicycling motion or carefully massaging their tummy to help stimulate their bowels.
+
+Give older children plenty of fluids and encourage them to eat fruit. Chop or puree it if it’s easier for them to eat. The best fruits for constipation include apples, grapes, pears and strawberries.
+
+Don’t force your child to eat as this can make mealtimes stressful.
+
+[What to feed young children](http://www.nhs.uk/Conditions/pregnancy-and-baby/Pages/understanding-food-groups.aspx).
+
+### Helping your child with potty training
+
+Some children feel anxious or stressed about using the toilet. This can cause them to hold in their poo and lead to constipation.
+
+This usually happens during potty training or if their usual toilet routine has changed. For example, after moving house or starting nursery.
+
+Give your child plenty of time to use the toilet while they are still learning. Encourage them when they do use the toilet. Some parents find a reward chart works.
+
+[Potty training tips](http://www.nhs.uk/conditions/pregnancy-and-baby/pages/potty-training-tips.aspx)
+
+
+## Your pharmacist can help with constipation
+
+Speak to your pharmacist if diet and lifestyle changes aren’t helping. They can suggest a suitable laxative. These are medicines that help you poo more regularly.
+
+Most laxatives work within 3 days. They should only be used for a short time.
+
+Laxatives are not recommended for children unless they are prescribed by your GP.
+
+[Different types of laxatives and how they work](https://www.nhs.uk/conditions/laxatives/Pages/Introduction.aspx).
```

Comment:
There are [numerous](http://writingspaces.org/wwsg/punctuation-hyperlinks) [suggestions](http://www.businesswritingblog.com/business_writing/2006/06/how_to_urls_and.html) (online)[https://lorelleteaches.com/2013/01/08/what-is-a-properly-formed-link/] to not include terminal punctuation in links.

Pretty much any online resource I can think of (including W3C, BBC, The Guardian, GOV.UK place punctuation outside of links.